int16 usB
int16 usL
int16 usR
int16 per40
int16 per12
int16 arduinoEStop
